pkg/server/transform.go

```diff
@@ -212,3 +217,72 @@ func translationFromProto(t *translatev1.Translation) (*model.Translation, error
 func translationsToProto(m []model.Translation) []*translatev1.Translation {
 	return sliceToProto(m, translationToProto)
 }
+
+// ----------------------Mask----------------------
+
+func maskFromProto(message proto.Message, mask *fieldmaskpb.FieldMask) (model.Mask, error) {
+	if mask == nil {
+		return nil, nil
+	}
+
+	paths := mask.GetPaths()
+
+	if len(paths) == 0 {
+		return model.Mask{}, nil
+	}
+
+	// Find corresponding model type to proto message.
+	var modelType any
```

Comment:
You should be able to use protoreflect and avoid explicit type implementation. Right? It is less expensive too.

```go
m.Protoreflect().Descriptor()
```

Reply:
okay, i understand what you meant, but still wont work as some fields differ 

proto generated Message, has `PluralId` when we in model we have `PluralID`...
<img width="286" alt="image" src="https://github.com/translate-agent/translate/assets/69954124/5ef03913-866f-4d46-a8e4-5c1d3bca4dec">

Reply:
same for `Id` In the future those differences will only increase in size

Reply:
The names are identical, but the case may differ.

Reply:
> Make it case-insensitive?

I did not think of that. That actually might and should work, and that would simplify everything a lot 👍

Reply:
Resolving edge cases is a big no-no. It is now hard to work on the project (somehow), and adding these workarounds will only make situation worse